id/news/_posts/2019-10-01-http-response-splitting-in-webrick-cve-2019-16254.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: news_post
+title: "CVE-2019-16254: Pemisahan respons HTTP pada WEBrick (Perbaikan tambahan)"
+author: "mame"
+translator: "meisyal"
+date: 2019-10-01 11:00:00 +0000
+tags: security
+lang: id
+---
+
+Ada sebuah kerentanan pemisahan respons HTTP pada WEBrick yang di-*bundle*
```

Comment:
Can we change di-bundle with **dibendel**?

Reply:
@gozali according to KBBI, *bendel* is not a valid word. KBBI suggests to use *bundel* instead. What do you think?

bendel: https://kbbi.kemdikbud.go.id/entri/bendel
bundel: https://kbbi.kemdikbud.go.id/entri/bundel%20(1)

Reply:
OK dibundel it is 👍

Reply:
updated to dibundel @gozali